airflow/configuration.py

```diff
@@ -41,6 +41,22 @@ def expand_env_var(env_var):
         else:
             env_var = interpolated
 
+def run_bash_command(command):
```

Comment:
please move this function to `airflow/utils.py`, I think by default it's just plain old `sh`. maybe rerun `run_command`